workspace-grid@mathematical.coffee.gmail.com/metadata.json

```diff
@@ -6,7 +6,7 @@
  "gettext-domain": "workspace-grid",
  "description": "Arranges workspaces in a configurable grid.\nAlso:\n* implements keybindings for left/right workspace navigation (up/down are already implemented)\n* updates workspaces sidebar with grid configuration (use Remove Workspaces Sidebar if you don't want it).\n\nYou may also wish to consider instead of/with this extension: Frippery Bottom Panel (already has workspace grid functionality if you want a bottom panel); Frippery Static Workspaces (holds the number of workspaces static and skips all the workspace grid stuff); Remove Workspaces Sidebar; Workspace Indicator Extension (textual indicator/workspace switcher); WorkspaceBar (graphical indicator/workspace switcher); Workspace navigator extension (use arrow keys to navigate workspaces in the Overview).\nSee homepage for more details.",
  "shell-version": [
-     "3.28"
```

Comment:
Why this breaks compatibility with 3.28?

Reply:
no it doesn't. Sorry I thought it was the "supported" version. Does that mean minimal supported version? I can revert this change of course.

Reply:
No this is supposed to contain all the supported versions, e.g. in the recent past it used to be

```
 "shell-version": [
     "3.16", "3.18", "3.20", "3.22", "3.24", "3.26"
],
```

but then 3.28 broke compatibility so the latest version currently only supports 3.28, so if this PR enables support for another version (not sure if intermediate versions like 3.29 count) please add it to the list

Reply:
@zakkak : thank you, I just added "3.30"!